### DISCUSSION TITLE
Does thorium Android support Google sync?

### DISCUSSION THREAD

**Discussion by Aedon** [Alex313031/Thorium-Android#4] [Q&A] [answered]
Been using thorium Android for a few weeks, but I'm unable to login to my Google account: (This account already exists on your device)
![Screenshot_2023-09-07-10-15-23-11_f7aa348215f5d566f9e4ca860f474209](https://github.com/Alex313031/Thorium-Android/assets/72438130/b3830659-2e70-495e-aba6-d0976621dc58)

**Comment by gz83** [comment, marked as answer]
[bugs.chromium.org/p/chromium/issues/detail?id=677887](https://bugs.chromium.org/p/chromium/issues/detail?id=677887)

Google restricts login and synchronization of third-party versions, and there is currently no solution

@Aedon

**Reply by Somnius** [reply]
It works flawlessly on windows builds, why not being able to login over android 😐😥😔

**Comment by vaclavbenes** [comment]
Stupid question: Is this not working due api keys ? it it possible add them to android ?

**Reply by gz83** [reply]
This is due to Google's limitations, unless Thorium manages to use a non-Google sync service.
